CHANGELOG/CHANGELOG-v0.4.md

```diff
@@ -0,0 +1,23 @@
+# Changelog since v0.3.0
+All notable changes to this project will be documented in this file.
+
+## v0.4.0
+
+### New Features 
+* Support managedClusterAddon registration.
```

Comment:
Support managedClusterAddon registration and maintain the lease of managedClusterAddon

Reply:
done